flow-typed/types.js

```diff
@@ -196,3 +196,12 @@ declare type Tenant = {
   jwtIssuer: string,
   jwksInstance: JwksClient,
 }
+
+declare type SetSendingAddressResponse = {
```

Comment:
There's already a type very similar to this in the types repo `flow-typed/forms-apps.js` called `FormsAppSendingAddress`. Think it will work here.